pkgs/applications/office/libreoffice/default-gen-shell.nix

```diff
@@ -19,9 +19,11 @@ stdenv.mkDerivation {
     builder = ./download-list-builder.sh;
   };
 
+  buildInputs = [ python36 ];
+
   shellHook = ''
     function generate {
-      ./generate-libreoffice-srcs.sh | tee libreoffice-srcs.nix
+      python3 generate-libreoffice-srcs.py > libreoffice-srcs.nix
```

Comment:
I agree - there's no harm in adding a shebang. I'd still leave the explicit call to `python3` in this line though.